numpy/core/einsumfunc.py

```diff
@@ -884,43 +893,73 @@ def einsum(*operands, **kwargs):
     -----
     .. versionadded:: 1.6.0
 
+    The Einstein summation convention can be used to compute
+    many multi-dimensional, linear algebraic array operations. `einsum`
+    provides a succinct way of representing these in a
+    clear and consistent format.
```

Comment:
"in a clear and consistent format" is a bit judgemental and not needed. Could also be read as "if you don't grok the syntax it is your fault"

Reply:
The general point I was trying to convey is that for algorithms that contain lots of different linear algebraic operations it can be clearer where this format is repetitively used.
But I also maintain the view that less verbose descriptions are generally better so I think you're right!